study-builder/studies/testboston/study-events.conf

```diff
@@ -219,6 +219,7 @@
       },
       # If there is already a Baseline Symptom survey, then this is after the initial kit.
       "cancelExpr": """user.studies["testboston"].forms["BASELINE_SYMPTOM"].hasInstance()""",
+      "maxOccurrencesPerUser": 1,
```

Comment:
This change was from a previous manual change in production. I don't remember which ticket it came from, but the idea is that this email should be sent only once per user.